lumicks/pylake/force_calibration/calibration_models.py

```diff
@@ -828,18 +838,19 @@ def __init__(
         )
         self.driving_frequency_guess = driving_frequency_guess
         self.sample_rate = sample_rate
-        self.num_windows = num_windows
+        self.num_windows = num_windows  # caveat: misnamed parameter
```

Comment:
perhaps mark this with `# todo: ` instead of `caveat`? I'll sometimes grep for `todo` to see if there's things to pick up